packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -796,6 +905,8 @@ abstract class Element implements BuildContext {
     _slot = newSlot;
     _depth = _parent != null ? _parent.depth + 1 : 1;
     _active = true;
+    if (parent != null) // Only assign ownership if the parent is non-null
+      _owner = parent.owner;
```

Comment:
I was trying to avoid forcing an ordering to lines 168 and 169 in widgets/binding.dart.

Now that I take a second look at this, though, I'm wondering if it would be easier just to pass the owner in the constructor. In that case, the assertion would be that if the parent is null, the owner field should already be non-null.